go/obscuronode/host/p2p/local_p2p.go

```diff
@@ -0,0 +1,124 @@
+package p2p
+
+import (
+	"fmt"
+
+	"github.com/ethereum/go-ethereum/rlp"
+	"github.com/obscuronet/obscuro-playground/go/log"
+	"github.com/obscuronet/obscuro-playground/go/obscurocommon"
+	"github.com/obscuronet/obscuro-playground/go/obscuronode/nodecommon"
+)
+
+// fake multicast network
+var networkLayer = make(map[string]chan []byte)
+
+// localP2PImpl implements the P2P in a channel based approach
+type localP2PImpl struct {
+	address       string
+	peerAddresses []string
+	txChan        chan nodecommon.EncryptedTx
+	rollupChan    chan obscurocommon.EncodedRollup
+}
+
+// NewLocalP2P returns a new P2P object that runs locally using channels.
+func NewLocalP2P(ourAddress string, allAddresses []string) P2P {
+	// We filter out our P2P address if it's contained in the list of all P2P addresses.
+	var peerAddresses []string
+	for _, a := range allAddresses {
+		if a != ourAddress {
+			peerAddresses = append(peerAddresses, a)
+		}
+	}
+
+	// ensure all network chans have been created
```

Comment:
This comment confused me. Can we change it to, "ensure our network chan has been created"?

Reply:
Good catch, it was an old comment 👍